Formula/git-xargs.rb

```diff
@@ -0,0 +1,21 @@
+class GitXargs < Formula
+  desc "CLI for making updates across multiple Github repositories with a single command"
+  homepage "https://github.com/gruntwork-io/git-xargs"
+  url "https://github.com/gruntwork-io/git-xargs/archive/v0.0.10.tar.gz"
+  sha256 "890e2105ef62117849fdf740d753b6b436b2989a108054913ebeb6b0d93218eb"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X main.VERSION=v#{version}")
+  end
+
+  test do
```

Comment:
Let's check the version here to e to make sure the `ldflags` we use are still valid.

Reply:
Sure